apps/bfd-pipeline/bfd-pipeline-ccw-rif/src/test/java/gov/cms/bfd/pipeline/ccw/rif/load/RifLoaderIT.java

```diff
@@ -818,6 +818,132 @@ public void loadBeneficiaryWhenUpdateAndNon2022EnrollmentDateAndFilterOffExpectR
     validateBeneficiaryAndSkippedCountsInDatabase(1, 0);
   }
 
+  /**
+   * Runs {@link RifLoader} against the {@link StaticRifResourceGroup#SAMPLE_A} data when INSERT,
+   * filter is on, and the LoadStrategy.INSERT_IDEMPOTENT is used with a non-Beneficiary type,
+   * expect the data is loaded normally.
+   */
+  @Test
+  public void loadNonBeneficiaryWhenInsertAndFilterOnAndInsertStrategyExpectRecordLoaded() {
+
+    loadDefaultSampleABeneData(CcwRifLoadTestUtils.getLoadOptions());
+    Stream<RifFile> stream =
+        filterSamples(
+            r -> r.getFileType() == RifFileType.INPATIENT,
+            StaticRifResourceGroup.SAMPLE_A.getResources());
+    loadSample(
+        "non-Bene sample",
+        CcwRifLoadTestUtils.getLoadOptionsWithFilteringofNon2022BenesEnabled(true),
+        stream);
+    validateBeneficiaryAndSkippedCountsInDatabase(1, 0);
+  }
+
+  /**
+   * Runs {@link RifLoader} against the {@link StaticRifResourceGroup#SAMPLE_A} data when INSERT,
+   * filter is on, and a non-Beneficiary type, expect the data is loaded normally.
+   */
+  @Test
+  public void loadNonBeneficiaryWhenInsertAndFilterOnExpectRecordLoaded() {
+
+    loadDefaultSampleABeneData(CcwRifLoadTestUtils.getLoadOptions());
+    Stream<RifFile> stream =
+        filterSamples(
+            r -> r.getFileType() == RifFileType.INPATIENT,
+            StaticRifResourceGroup.SAMPLE_A.getResources());
+    loadSample(
+        "non-Bene sample",
+        CcwRifLoadTestUtils.getLoadOptionsWithFilteringofNon2022BenesEnabled(false),
+        stream);
+    validateBeneficiaryAndSkippedCountsInDatabase(1, 0);
+  }
+
+  /**
+   * Runs {@link RifLoader} against the {@link StaticRifResourceGroup#SAMPLE_A} data when INSERT,
+   * filter is off, and the LoadStrategy.INSERT_IDEMPOTENT is used with a non-Beneficiary type,
+   * expect the data is loaded normally.
+   */
+  @Test
+  public void loadNonBeneficiaryWhenInsertAndFilterOffAndInsertStrategyExpectRecordLoaded() {
+
+    loadDefaultSampleABeneData(CcwRifLoadTestUtils.getLoadOptions());
+    Stream<RifFile> stream =
+        filterSamples(
+            r -> r.getFileType() == RifFileType.INPATIENT,
+            StaticRifResourceGroup.SAMPLE_A.getResources());
+    loadSample("non-Bene sample", CcwRifLoadTestUtils.getLoadOptions(), stream);
+    validateBeneficiaryAndSkippedCountsInDatabase(1, 0);
+  }
+
+  /**
+   * Runs {@link RifLoader} against the {@link StaticRifResourceGroup#SAMPLE_A} data when INSERT,
+   * filter is off, and a non-Beneficiary type, expect the data is loaded normally.
+   */
+  @Test
+  public void loadNonBeneficiaryWhenInsertAndFilterOffExpectRecordLoaded() {
```

Comment:
I think a lot of these new test cases with filtering turned off are duplicative of the OG `loadSampleA` test.

Exceptions to this are any that are verifying that exceptions ARE thrown.

Reply:
Good point, added a check to the sample A test and removed the tests which were not doing anything unique here.